src/app/workbasket/workbasket.component.html

```diff
@@ -3,9 +3,9 @@
 
     <div class="display-left" leftBody>
       <ccd-workbasket-filters
```

Comment:
Seems like after this is merge we will be using the new contract in man web. This seems wrong to me as we will be making a call for data we already have in `profile.jurisdictions`.

Reply:
This has now been regressed.